From: Prarit Bhargava <prarit@redhat.com>
Date: Fri, 8 Feb 2019 19:24:49 +0100
Subject: printk: Export console_printk
Git-commit: a1939185c7a907f91b40da259a610ce2e2da9e18
Patch-mainline: v5.1-rc1
References: bsc#1071995 fate#323487

The fbcon can be built as a module and requires console_printk.

Export console_printk.

Signed-off-by: Prarit Bhargava <prarit@redhat.com>
Reviewed-by: Sergey Senozhatsky <sergey.senozhatsky@gmail.com>
Acked-by: Petr Mladek <pmladek@suse.com>
Signed-off-by: Bartlomiej Zolnierkiewicz <b.zolnierkie@samsung.com>
Acked-by: Miroslav Benes <mbenes@suse.cz>
---
 kernel/printk/printk.c |    1 +
 1 file changed, 1 insertion(+)

--- a/kernel/printk/printk.c
+++ b/kernel/printk/printk.c
@@ -65,6 +65,7 @@ int console_printk[4] = {
 	CONSOLE_LOGLEVEL_MIN,		/* minimum_console_loglevel */
 	CONSOLE_LOGLEVEL_DEFAULT,	/* default_console_loglevel */
 };
+EXPORT_SYMBOL_GPL(console_printk);
 
 /*
  * Low level drivers may need that to know if they can schedule in
